index.html

```diff
@@ -30,5 +30,82 @@ <h2>I’am a Software Developer</h2>
         <div class="illustration"><img src="images/illustration-header-mobile.svg" alt="illustration"></div>
     </div>
 </section>
+<div class="works">
```

Comment:
Kindly. each part of your website should be a SECTION tag, this will give a better structure to your site